src/Internal.cc

```diff
@@ -51,4 +55,43 @@ namespace cbl_internal {
         return json;
     }
 
+#ifdef __ANDROID__
+
+    static CBLInitContext sInitContext;
+    static std::mutex sInitMutex;
```

Comment:
I think this is overkill given its use case.  The user is going to be calling this once early in the lifecycle, almost certainly long before doing anything multithreaded.

Reply:
I could agree with your comment. I can remove the mutex.